### PR TITLE
InteractingScp330.cs: Reduction of bloat code from original design.

### PR DESCRIPTION
**Pull request by Undid-Iridium**
## Description
Same behavior, less bloat, except for the primary check:

 // Remove original "No scp can touch" logic.
            newInstructions.RemoveRange(0, 3);

	if (!ply.IsHuman() || ply.HasBlock(BlockedInteraction.GrabItems))
	{
		return;
	}

	
That has been returned, presuming the client won't even send the request if it's not a human. If this behavior should be re-introduced, just add the RemoveRange back.

**What is the current behavior?** (You can also link to an open issue here)
Works fine, redundant code - SCP changed since original design. 

**What is the new behavior?** (if this is a feature change)
In theory same behavior. 

**Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
No

**Other information**:

<br />

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentations
<br />

## Submission checklist

- [X] I have checked the project can be compiled
- [X] I have tested my changes and it worked as expected - side bar, even with previous behavior, if force re-spawning with RA, user loses hand on first attempt - bug? Unsure, behavior seems to be the same nonetheless.

### Patches (if there are any changes related to Harmony patches)
- [X] I have checked no IL patching errors in the console
- 
image

### Other
- [X] Still requires more testing - ¯\_(ツ)_/¯

If there is a desire to ensure nwapi behavior is kept:
`
if (playerInteractScp330Event.AllowPunishment && num2 >= 2)
`
Remove: `new(OpCodes.Br, enableEffect),`

Which will cause the the NW logic to occur. Otherwise, this will jump straight to

`ply.playerEffectsController.EnableEffect<SeveredHands>(0f, false);`

### CODE REVIEW

File: EXILED/Exiled.Events/Patches/Events/Scp330/InteractingScp330.cs [hunk 1]
```diff
@@ -45,9 +39,6 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
 
             LocalBuilder ev = generator.DeclareLocal(typeof(InteractingScp330EventArgs));
 
-            // Remove original "No scp can touch" logic.
-            newInstructions.RemoveRange(0, 3);
-
```
Comment (louis1706) [inline EXILED/Exiled.Events/Patches/Events/Scp330/InteractingScp330.cs:50]: This was a feature :3 then i made

Comment (Undid-Iridium) [reply EXILED/Exiled.Events/Patches/Events/Scp330/InteractingScp330.cs:50]: So - readd?

Comment (louis1706) [reply EXILED/Exiled.Events/Patches/Events/Scp330/InteractingScp330.cs:50]: `That has been returned, presuming the client won't even send the request if it's not a human. If this behavior should be re-introduced, just add the RemoveRange back.`

So add it back: 3

Comment (Undid-Iridium) [reply EXILED/Exiled.Events/Patches/Events/Scp330/InteractingScp330.cs:50]: 👍

**Review by louis1706 — APPROVED** [review]

**Comment by Vladislav-CS** [comment]
I don't really understand why it's needed, nothing will change after it

**Comment by Undid-Iridium** [comment]
> I don't really understand why it's needed, nothing will change after it

Easier to maintain ¯\_(ツ)_/¯

**Comment by Undid-Iridium** [comment]
Personally, I don't care whether this gets accepted or not, it's an offer `¯\_(ツ)_/¯`

**Comment by Vladislav-CS** [comment]
Well, yeah, my bad, I thought it only removes the check, but it doesn't

**Review by Vladislav-CS — APPROVED** [review]

**Pull request merged**